pages/collection/[chain]/[contract]/[id].tsx

```diff
@@ -109,6 +110,22 @@ const IndexPage: NextPage<Props> = ({ id, collectionId, ssr }) => {
 
   return (
     <Layout>
+      <Head>
+        <title>
+          {token?.token?.name
+            ? token.token.name
+            : `${token?.token?.tokenId} - ${token?.token?.collection}`}
+        </title>
+        <meta name="description" content={collection?.description as string} />
+        <meta
+          name="twitter:image"
+          content={token?.token?.image || collection?.banner}
+        />
+        <meta
+          property="og:image"
+          content={token?.token?.image || collection?.banner}
```

Comment:
Do we want to add the token name / token id to the og title?

Reply:
good call. I'll update that